packages/ui-test-sandbox/src/utils/sandbox.ts

```diff
@@ -27,11 +27,6 @@ import { ReactComponentWrapper } from './reactComponentWrapper'
 import initConsole from './initConsole'
 import React from 'react'
 
-// Likely this could be done with /// <reference types
-declare const before: any
-declare const beforeEach: any
-declare const afterEach: any
-
```

Comment:
These are not needed because TS is able to pick up the type definitions from 'mocha'.